spec/datadog/core/error_spec.rb

```diff
@@ -130,7 +130,7 @@ def call
             end
           end
 
-          it 'reports errors only once', if: (RUBY_VERSION < '2.6.0' || PlatformHelpers.truffleruby?) do
+          it 'reports errors only once', if: (RUBY_VERSION < '2.6.0' || PlatformHelpers.truffleruby? || PlatformHelpers.jruby? && RUBY_ENGINE_VERSION >= '9.3.7.0') do
```

Comment:
I suggest adding some parenthesis so people don't have to wonder about precedence ;)

```suggestion
          it 'reports errors only once', if: (RUBY_VERSION < '2.6.0' || PlatformHelpers.truffleruby? || (PlatformHelpers.jruby? && RUBY_ENGINE_VERSION >= '9.3.7.0')) do
```

Reply:
Add too much parentheses and Rubocop will complain :D

I would otherwise agree in general but I'm pretty sure most everyone knows the precedence between `||` and `&&` so it's quire readable.